Modules/Scaler/DialsScaler.py

```diff
@@ -89,6 +89,9 @@ def _updated_dials_scaler(self):
             PhilIndex.params.dials.scale.partiality_threshold
         )
 
+        if self.get_scaler_anomalous():
```

Comment:
Could oneline this by `self._scaler.set_anomalous(self.get_scaler_anomalous())`?